Lab3.ipynb

```diff
@@ -0,0 +1,429 @@
+      "text/plain": [
+       "<Figure size 1440x1080 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "a = np.vstack((stats.norm.rvs( scale = 1, size = 100000), stats.norm.rvs( scale = 1, size = 100000)))\n",
+    "a.shape\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "h = ax.hist2d(a[0,:],a[1,:],bins=100, density=True);\n",
+    "ax.set_aspect('equal', 'box')\n",
+    "plt.xlim([-3 , 3])\n",
+    "plt.ylim([-3 , 3])\n",
+    "plt.title(\"2D Histogram of positional uncertainty\", fontsize = 24)\n",
+    "plt.ylabel(\"$\\Delta$y arcseconds\", fontsize = 18)\n",
+    "plt.xlabel(\"$\\Delta$x arcseconds\", fontsize = 18)\n",
+    "plt.colorbar(h[3], ax=ax)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 2.1**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The statistical question I have is what is the displacement or movement to see a 5-sigma detection from the background in arcseconds? The reason this is my statistical question is because an asteroid will have some true movement between the images. The analytic pdf we are dealing with is a Rayleigh distribution. We would want to evaluate an integral of a Rayleigh distribution (with mean = 0, sigma = 1) from the probability to get 5-sigma to positive infinity to find the signal associated with 5-sigma detection. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Section 2.2**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the code, we have a 2D Gaussian distribution with 100k data points where mean is 0 and sigma is 1. We would want to find at what value in the distribution corresponds to a 5-sigma detection from the background. "
```

Comment:
I would also explain why you are using the Rayleigh in the next part. As well as mentions that we can turn this 2D dist into a 1D dist since we care about the magnitude of the displacement and not the x and y components alone.